apps/wing-console/console/server/src/router/test.ts

```diff
@@ -62,7 +63,11 @@ const runTest = async (
         messageType: "success",
       },
     );
-  } catch {
+  } catch (error: any) {
+    let output = await formatTraceError(error?.message);
```

Comment:
```suggestion
    const output = await formatTraceError(error?.message);
```

Getting used to wing ha?